src/export-transactions.ts

```diff
@@ -77,17 +76,17 @@ const FACILITATOR_ADDRESS = '0x76fb13f00cdbdd5eac8e2664cf14be791af87cb0'
 async function getTopicTxs(network: number, startblock: number, topic: Topic) {
   const events: string[] = []
   let block = startblock
-  let url = `https://api.covalenthq.com/v1/${network}/block_v2/latest/?key=${API_KEY}`
+  let url = `https://api.covalenthq.com/v1/${network}/block_v2/latest/?key=${COVALENT_API_KEY}`
   let json = await fetchURL(url)
   const latestBlock: number = json.data.items[0].height
   console.log('Latest', network, block, latestBlock, (latestBlock - block) / 1000000)
 
   while (block < latestBlock) {
-    url = `https://api.covalenthq.com/v1/${network}/events/topics/${topic}/?key=${API_KEY}&starting-block=${block}&ending-block=${block + 1000000}&page-size=1000000000`
+    url = `https://api.covalenthq.com/v1/${network}/events/topics/${topic}/?key=${COVALENT_API_KEY}&starting-block=${block}&ending-block=${block + 1000000}&page-size=1000000000`
     console.log('fetch', url)
     json = await fetchURL(url)
     const data: APIEvents = json.data
-    let evs = data.items.map(e => e.tx_hash)
+    const evs = data.items.map(e => e.tx_hash)
```

Comment:
I know it is not unrelated with your changes but... What is `evs`? Is it events? Maybe `eventsWithTxHash`?